utils/tokenization.py

```diff
@@ -62,7 +62,6 @@ def load_vocab(vocab_file):
 
 def whitespace_tokenize(text):
```

Comment:
This function from huggingface is absolutely hilarious because it's literally the same as `str.split`.

Reply:
If we are using TorchText, we would have to create a lambda with str.split() for tokenize parameter. This might be better in that case as we can reuse this.

Reply:
Also, whitespace_tokenize(None) would not raise an exception